src/components/Player.vue

```diff
@@ -238,20 +236,18 @@ export default {
         }
       }
     },
-    playing() {
+    async playing() {
       if (this.playing) {
-        this.$refs.audio
-          .play()
-          .then(() => {
-            if ("mediaSession" in navigator) {
-              navigator.mediaSession.playbackState = "playing";
-            }
-          })
-          .catch((error) => {
-            new Error(error);
-          });
+        try {
+          await this.$refs.audio.play();
+          if ("mediaSession" in navigator) {
+            navigator.mediaSession.playbackState = "playing";
+          }
+        } catch (error) {
+          new Error(error);
```

Comment:
This doesn't make any sense. (I realise this was already something that didn't make any sense in the code that existed before, that doesn't mean it shouldn't be fixed.)

Reply:
I've changed this to commit the error to the store, as we also do that when watching the `currentTrackURL`